recipes/zaber-motion/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+macos_min_version:             # [osx]
+  - 10.10                      # [osx]
```

Comment:
@scopatz this should rerender things correctly

Reply:
OK! I am happy for you to figure this out in a feedstock